### ISSUE TITLE
Bot gets stuck or randomly switches forts

### ISSUE THREAD

**Issue by jonathanbalagot** [OpenPoGo/OpenPoGoBot#148]
Not sure if this one was reported. When I enable use google_directions, it seems to eventually do one of the following

1) Walk to the fort then get stuck with repeating distance
2) After being stuck, switch fort, walk some distance then gets stuck with repeating distance

Commit - e1bd866e7297663bb236efd719a4122543a4adc5

```
[2016-07-31 23:58:38] [#] Now at Pokestop: The Pond
[2016-07-31 23:58:38] [PokeStop] Spinning...
[2016-07-31 23:58:41] [PokeStop] Loot:
[2016-07-31 23:58:41] [PokeStop] + 50 XP
[2016-07-31 23:58:41] [PokeStop] + 1 Revive
[2016-07-31 23:58:41] [PokeStop] + 1 Razz Berry
[2016-07-31 23:58:41] [PokeStop] + 2 Pokeballs
[2016-07-31 23:58:41] [PokeStop] PokeStop is on cooldown for 5.04 minutes.
[2016-07-31 23:58:42] [#] Found fort Subway 5th  at distance 0.12km
[2016-07-31 23:58:42] [#] Moving closer to Subway 5th
[2016-07-31 23:58:42] [#] Asking google for directions
[2016-07-31 23:58:42] [#] Walking from (40.7660345, -73.9757963) to (40.7655582, -73.9749664) for approx. 36.00 seconds
[2016-07-31 23:58:45] [#] Found fort Subway 5th  at distance 0.12km
[2016-07-31 23:58:45] [#] Moving closer to Subway 5th
[2016-07-31 23:58:45] [#] Asking google for directions
[2016-07-31 23:58:45] [#] Walking from (40.7660131, -73.9757848) to (40.7655582, -73.9749664) for approx. 35.00 seconds
[2016-07-31 23:58:49] [#] Found fort Subway 5th  at distance 0.12km
[2016-07-31 23:58:49] [#] Moving closer to Subway 5th
[2016-07-31 23:58:49] [#] Asking google for directions
[2016-07-31 23:58:49] [#] Walking from (40.7659376, -73.9757388) to (40.7655582, -73.9749664) for approx. 32.00 seconds
[2016-07-31 23:58:53] [#] Found fort Subway 5th  at distance 0.12km
[2016-07-31 23:58:53] [#] Moving closer to Subway 5th
[2016-07-31 23:58:53] [#] Asking google for directions
[2016-07-31 23:58:53] [#] Walking from (40.7658893, -73.97570189999999) to (40.7655582, -73.9749664) for approx. 29.00 seconds
[2016-07-31 23:58:58] [#] Found fort Subway 5th  at distance 0.12km
[2016-07-31 23:58:58] [#] Moving closer to Subway 5th
[2016-07-31 23:58:58] [#] Asking google for directions
[2016-07-31 23:58:58] [#] Walking from (40.7658601, -73.975673) to (40.7655582, -73.9749664) for approx. 28.00 seconds
[2016-07-31 23:59:03] [#] Found fort Subway 5th  at distance 0.12km
[2016-07-31 23:59:03] [#] Moving closer to Subway 5th
[2016-07-31 23:59:03] [#] Asking google for directions
[2016-07-31 23:59:03] [#] Walking from (40.7658195, -73.97562959999999) to (40.7655582, -73.9749664) for approx. 26.00 seconds
[2016-07-31 23:59:06] [#] Found fort Subway 5th  at distance 0.12km
[2016-07-31 23:59:06] [#] Moving closer to Subway 5th
[2016-07-31 23:59:06] [#] Asking google for directions
[2016-07-31 23:59:06] [#] Walking from (40.76572360000001, -73.9755119) to (40.7655582, -73.9749664) for approx. 20.00 seconds
[2016-07-31 23:59:10] [#] Found fort Subway 5th  at distance 0.12km
[2016-07-31 23:59:10] [#] Moving closer to Subway 5th
[2016-07-31 23:59:10] [#] Asking google for directions
[2016-07-31 23:59:10] [#] Walking from (40.7656752, -73.9753763) to (40.7655582, -73.9749664) for approx. 15.00 seconds
[2016-07-31 23:59:14] [#] Found fort Subway 5th  at distance 0.12km
[2016-07-31 23:59:14] [#] Moving closer to Subway 5th
[2016-07-31 23:59:14] [#] Asking google for directions
[2016-07-31 23:59:15] [#] Walking from (40.7656443, -73.9752901) to (40.7655582, -73.9749664) for approx. 12.00 seconds
[2016-07-31 23:59:19] [#] Found fort Subway 5th  at distance 0.12km
[2016-07-31 23:59:19] [#] Moving closer to Subway 5th
[2016-07-31 23:59:19] [#] Asking google for directions
[2016-07-31 23:59:20] [#] Walking from (40.765624, -73.9752333) to (40.7655582, -73.9749664) for approx. 10.00 seconds
[2016-07-31 23:59:23] [#] Found fort Sweeny Memorial Bench at distance 0.07km
[2016-07-31 23:59:23] [#] Moving closer to Sweeny Memorial Bench
[2016-07-31 23:59:23] [#] Asking google for directions
[2016-07-31 23:59:24] [#] Walking from (40.7655991, -73.9751635) to (40.766133, -73.9758098) for approx. 33.00 seconds
[2016-07-31 23:59:26] [#] Found fort Sweeny Memorial Bench at distance 0.07km
[2016-07-31 23:59:26] [#] Moving closer to Sweeny Memorial Bench
[2016-07-31 23:59:26] [#] Asking google for directions
[2016-07-31 23:59:27] [#] Walking from (40.7656196, -73.97522099999999) to (40.766133, -73.9758098) for approx. 31.00 seconds
[2016-07-31 23:59:32] [#] Found fort Sweeny Memorial Bench at distance 0.07km
[2016-07-31 23:59:32] [#] Moving closer to Sweeny Memorial Bench
[2016-07-31 23:59:32] [#] Asking google for directions
[2016-07-31 23:59:33] [#] Walking from (40.7656643, -73.9753461) to (40.766133, -73.9758098) for approx. 27.00 seconds
[2016-07-31 23:59:36] [#] Found fort Sweeny Memorial Bench at distance 0.07km
[2016-07-31 23:59:36] [#] Moving closer to Sweeny Memorial Bench
[2016-07-31 23:59:36] [#] Asking google for directions
[2016-07-31 23:59:36] [#] Walking from (40.7656725, -73.975369) to (40.766133, -73.9758098) for approx. 26.00 seconds
[2016-07-31 23:59:40] [#] Found fort Sweeny Memorial Bench at distance 0.07km
[2016-07-31 23:59:40] [#] Moving closer to Sweeny Memorial Bench
[2016-07-31 23:59:40] [#] Asking google for directions
[2016-07-31 23:59:40] [#] Walking from (40.765771, -73.97557069999999) to (40.766133, -73.9758098) for approx. 19.00 seconds
[2016-07-31 23:59:45] [#] Found fort Sweeny Memorial Bench at distance 0.07km
[2016-07-31 23:59:45] [#] Moving closer to Sweeny Memorial Bench
[2016-07-31 23:59:45] [#] Asking google for directions
[2016-07-31 23:59:45] [#] Walking from (40.7658539, -73.9756669) to (40.766133, -73.9758098) for approx. 14.00 seconds
[2016-07-31 23:59:48] [#] Found fort Sweeny Memorial Bench at distance 0.07km
[2016-07-31 23:59:48] [#] Moving closer to Sweeny Memorial Bench
[2016-07-31 23:59:48] [#] Asking google for directions
[2016-07-31 23:59:48] [#] Walking from (40.766133, -73.9758098) to (40.7660851, -73.9761771) for approx. 13.00 seconds
[2016-07-31 23:59:53] [#] Found fort Sweeny Memorial Bench at distance 0.07km
[2016-07-31 23:59:53] [#] Moving closer to Sweeny Memorial Bench
[2016-07-31 23:59:53] [#] Asking google for directions
[2016-07-31 23:59:53] [#] Walking from (40.766133, -73.9758098) to (40.7660851, -73.9761771) for approx. 13.00 seconds
[2016-07-31 23:59:57] [#] Found fort Sweeny Memorial Bench at distance 0.07km
[2016-07-31 23:59:57] [#] Moving closer to Sweeny Memorial Bench
[2016-07-31 23:59:57] [#] Asking google for directions
[2016-07-31 23:59:57] [#] Walking from (40.766133, -73.9758098) to (40.7660851, -73.9761771) for approx. 13.00 seconds
[2016-08-01 00:00:00] [#] Found fort Sweeny Memorial Bench at distance 0.07km
[2016-08-01 00:00:00] [#] Moving closer to Sweeny Memorial Bench
[2016-08-01 00:00:00] [#] Asking google for directions
[2016-08-01 00:00:00] [#] Walking from (40.766133, -73.9758098) to (40.7660851, -73.9761771) for approx. 13.00 seconds
[2016-08-01 00:00:05] [#] Found fort Sweeny Memorial Bench at distance 0.07km
[2016-08-01 00:00:05] [#] Moving closer to Sweeny Memorial Bench
[2016-08-01 00:00:05] [#] Asking google for directions
[2016-08-01 00:00:05] [#] Walking from (40.766133, -73.9758098) to (40.7660851, -73.9761771) for approx. 13.00 seconds
[2016-08-01 00:00:09] [#] Found fort Sweeny Memorial Bench at distance 0.07km
[2016-08-01 00:00:09] [#] Moving closer to Sweeny Memorial Bench
[2016-08-01 00:00:09] [#] Asking google for directions
[2016-08-01 00:00:10] [#] Walking from (40.766133, -73.9758098) to (40.7660851, -73.9761771) for approx. 13.00 seconds
[2016-08-01 00:00:15] [#] Found fort Sweeny Memorial Bench at distance 0.07km
[2016-08-01 00:00:15] [#] Moving closer to Sweeny Memorial Bench
[2016-08-01 00:00:15] [#] Asking google for directions
[2016-08-01 00:00:15] [#] Walking from (40.7661422, -73.975843) to (40.7660851, -73.9761771) for approx. 12.00 seconds
[2016-08-01 00:00:19] [#] Found fort Sweeny Memorial Bench at distance 0.07km
[2016-08-01 00:00:19] [#] Moving closer to Sweeny Memorial Bench
[2016-08-01 00:00:19] [#] Asking google for directions
[2016-08-01 00:00:19] [#] Walking from (40.7661543, -73.9759443) to (40.7660851, -73.9761771) for approx. 9.00 seconds
[2016-08-01 00:00:24] [#] Found fort Sweeny Memorial Bench at distance 0.07km
[2016-08-01 00:00:24] [#] Moving closer to Sweeny Memorial Bench
[2016-08-01 00:00:24] [#] Asking google for directions
[2016-08-01 00:00:24] [#] Walking from (40.7661525, -73.9760482) to (40.7660851, -73.9761771) for approx. 6.00 seconds
[2016-08-01 00:00:29] [#] Found fort Sweeny Memorial Bench at distance 0.07km
[2016-08-01 00:00:29] [#] Moving closer to Sweeny Memorial Bench
[2016-08-01 00:00:29] [#] Asking google for directions
[2016-08-01 00:00:29] [#] Walking from (40.7660306, -73.9761389) to (40.7662158, -73.9763348) for approx. 11.00 seconds
[2016-08-01 00:00:34] [#] Found fort Sweeny Memorial Bench at distance 0.07km
[2016-08-01 00:00:34] [#] Moving closer to Sweeny Memorial Bench
[2016-08-01 00:00:34] [#] Asking google for directions
[2016-08-01 00:00:34] [#] Walking from (40.7661256, -73.9761098) to (40.7660851, -73.9761771) for approx. 3.00 seconds
[2016-08-01 00:00:35] [#] Walking from (40.7660851, -73.9761771) to (40.7662158, -73.9763348) for approx. 8.00 seconds
[2016-08-01 00:00:37] [#] Found fort Sweeny Memorial Bench at distance 0.07km
[2016-08-01 00:00:37] [#] Moving closer to Sweeny Memorial Bench
[2016-08-01 00:00:37] [#] Asking google for directions
[2016-08-01 00:00:37] [#] Walking from (40.7661209, -73.9762203) to (40.7662158, -73.9763348) for approx. 6.00 seconds
[2016-08-01 00:00:40] [#] Walking from (40.7662158, -73.9763348) to (40.7661399, -73.9763694) for approx. 4.00 seconds
[2016-08-01 00:00:41] [#] Found fort Sweeny Memorial Bench at distance 0.07km
[2016-08-01 00:00:41] [#] Moving closer to Sweeny Memorial Bench
[2016-08-01 00:00:41] [#] Asking google for directions
[2016-08-01 00:00:41] [#] Walking from (40.7661938, -73.9763083) to (40.7662158, -73.9763348) for approx. 2.00 seconds
[2016-08-01 00:00:41] [#] Walking from (40.7662158, -73.9763348) to (40.7661399, -73.9763694) for approx. 4.00 seconds
[2016-08-01 00:00:43] [#] Walking from (40.7661399, -73.9763694) to (40.76631829999999, -73.9766798) for approx. 14.00 seconds
[2016-08-01 00:00:45] [#] Found fort Sweeny Memorial Bench at distance 0.07km
[2016-08-01 00:00:45] [#] Moving closer to Sweeny Memorial Bench
[2016-08-01 00:00:45] [#] Asking google for directions
[2016-08-01 00:00:45] [#] Walking from (40.7661399, -73.9763694) to (40.76631829999999, -73.9766798) for approx. 14.00 seconds
```

**Comment by wchill** [comment]
This might be related to #147.

**Comment by andythorne** [comment]
Fixed in #136
